modules/installation-vsphere-config-yaml.adoc

```diff
@@ -99,6 +99,10 @@ imageContentSources: <18>
   source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
 endif::openshift-origin[]
 endif::restricted[]
+capabilities:
+  baselineCapabilitySet: None
+  additionalEnabledCapabilities:
+  - openshift-samples
```

Comment:
I found that there is no optional parameters table in UPI installation docs, but have such example of capabilities without any explanation, if user directly read those docs, they may confused about that.

Reply:
Thank you Jinyun. I see that the UPI docs are not including the configuration parameter tables -- this may be a bug. I will open a BZ to investigate, but that is outside of the scope of this PR. Does the rest of the PR look good to you?

Reply:
thanks for addressing it, then it looks good to me.